CONTRIBUTING.md

```diff
@@ -11,47 +11,38 @@ The documentation is built using [Sphinx](http://sphinx-doc.org) and [reStructur
 
 Once you have cloned the Docs to your local machine, the following instructions will walk you through installing the tools necessary to build and test.
 
-1. [Download python](https://www.python.org/downloads/) version 2.7.10 or higher (Version 3.4 is recommended).
+1. [Download python](https://www.python.org/downloads/) version 2.7.10 or higher (Version 3.5 is recommended). If you install 3.5, check the include python in path option.
 
-2. If you are installing on Windows, ensure both the Python install directory and the Python scripts directory have been added to your `PATH` environment variable. For example, if you install Python into the c:\python34 directory, you would add `c:\python34;c:\python34\scripts` to your `PATH` environment variable.
+2. Skip this step if you installed python 3.5 and checked the "include in python path" option. If you are installing on Windows, ensure both the Python install directory and the Python scripts directory have been added to your `PATH` environment variable. For example, if you install Python into the c:\python34 directory, you would add `c:\python34;c:\python34\scripts` to your `PATH` environment variable.
 
-3. Install Sphinx by opening a command prompt and running the following Python command. (Note that this operation might take a few minutes to complete.)
+3. Clone the docs: 
 
-    ```pip install sphinx```
-
-4. By default, when you install Sphinx, it will install the ReadTheDocs custom theme automatically. If you need to update the installed version of this theme, you should run:
-
-    ```pip install -U sphinx_rtd_theme```
-
-5. Install the Sphinx .NET domain:
-
-    ```pip install sphinxcontrib-dotnetdomain```
-    
-6. Navigate to one of the main project subdirectories in the Docs repo - such as `mvc`, `aspnet`, or `webhooks`.
+    ```git clone https://github.com/aspnet/Docs.git```
+ 
+4. Navigate to the Docs repo and install the build dependencies:
```

Comment:
I might suggest putting in the actual commands beneath it helps with new OSS contributors and in this case I had to do a `ls` or `dir` to find out where the `aspnet` folder was.
```
cd aspnet
```